.github/workflows/publish.yml

```diff
@@ -102,48 +94,28 @@ jobs:
 
       - name: Update completed targets and remove label
         if: ${{ cancelled() || failure() }}
-        uses: actions/github-script@v5
         env:
           PUBLISH_ARGS: ${{ steps.inputs.outputs.result }}
-        with:
-          script: |
-            const Sentry = require('@sentry/node');
-            const inputs = JSON.parse(process.env.PUBLISH_ARGS);
-            const updateIssue = require(`${process.env.GITHUB_WORKSPACE}/.__publish__/src/publish/update-issue.js`).default;
-            return await updateIssue({context, github, inputs});
+          GITHUB_TOKEN: ${{ github.token }}
+        run: node .__publish__/src/publish/update-issue.js
 
       - name: Inform about cancellation
         if: ${{ cancelled() }}
-        uses: actions/github-script@v5
         env:
           PUBLISH_ARGS: ${{ steps.inputs.outputs.result }}
-        with:
-          script: |
-            const Sentry = require('@sentry/node');
-            const inputs = JSON.parse(process.env.PUBLISH_ARGS);
-            const cancelled = require(`${process.env.GITHUB_WORKSPACE}/.__publish__/src/publish/cancelled.js`).default;
-            return await cancelled({context, github, inputs, Sentry});
+          GITHUB_TOKEN: ${{ github.token }}
```

Comment:
`github.token` and `secrets.GITHUB_TOKEN` are the same -- though I tend to use the latter since it's ever-so-slightly more clear that it's a privileged token